alma/endpoints/orders.py

```diff
@@ -13,10 +13,10 @@ def update(self, order_id, data):
         return Order(response.json)
 
     def fetch_all(self, limit: int = 20, starting_after: str = None, **filters):
-        args = {"limit": limit}
+        args = {"limit": str(limit)}
 
         if starting_after:
-            args["starting_after"] = starting_after
+            args["starting_after"] = str(starting_after)
```

Comment:
Not sure I get the need for that change? `starting_after` is already annotated as a `str`, and cannot be `None` in that code branch

Reply:
Good catch!, I guess mypy was puzzled to have limit as an int and starting_after as a str.